service/src/main/java/edu/tamu/catalog/controller/CatalogAccessController.java

```diff
@@ -64,6 +68,24 @@ public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="e
         }
     }
 
+    /**
+     * Provides data for all fees and fines associated with a patron.
+     *
+     * @param String catalogName (optional)
+     * @param String user
+     * @return
+     */
+    @GetMapping("/fines")
+    public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="folio") String catalogName, @WeaverUser User user) {
+        FeesFines FeesFines = getCatalogServiceByName(catalogName).getFeesFines(user.getUsername());
+
+        if (FeesFines != null) {
+            return new ApiResponse(SUCCESS, FeesFines);
+        } else {
+            return new ApiResponse(ERROR, "Error retrieving fines from " + catalogName + " catalog");
```

Comment:
This is one reason ApiResponse is a bad REST response. We should be returning a meaningful response code such as not found or some other appropriate error code.